README.md

```diff
@@ -0,0 +1 @@
+This is an exerise to build a photo repository with Django
```

Comment:
Add a descriptive readme, explain how a user would set up the repo, describe the models used etc.